Packs/HelloWorld/Integrations/HelloWorldEventCollector/HelloWorldEventCollector.py

```diff
@@ -209,7 +220,9 @@ def main() -> None:  # pragma: no cover
 
             add_time_to_events(events)
             send_events_to_xsiam(events, vendor=VENDOR, product=PRODUCT)
```

Comment:
Before this line we can do also something like: 
demisto.debug(f"Sending {len(events)} events.")